packages/components/src/tooltip/index.js

```diff
@@ -24,7 +24,65 @@ import Shortcut from '../shortcut';
  *
  * @type {number}
  */
-const TOOLTIP_DELAY = 700;
+export const TOOLTIP_DELAY = 700;
+
+const eventCather = (
+	<div
+		className="event-catcher"
+		style={ {
+			zIndex: 9999,
```

Comment:
We have afaik a z-index management SCSS file with variables instead if hardcoded values.